platforms/ide/tooling-api/src/crossVersionTest/groovy/org/gradle/integtests/tooling/r930/ResilientKotlinDslScriptsModelBuilderCrossVersionSpec.groovy

```diff
@@ -649,13 +647,13 @@ class ResilientKotlinDslScriptsModelBuilderCrossVersionSpec extends ToolingApiSp
 
         when:
         settingsPlugin << """ broken !!! """
-        fails {
+        def mod = succeeds {
             action(KotlinModelAction.resilientModel(ROOT_PROJECT_FIRST)).withArguments("-Dorg.gradle.internal.resilient-model-building=true").run()
         }
 
         then:
-        def e = thrown(GradleConnectionException)
-        e.message.startsWith("The supplied build action failed with an exception.")
+        mod.failures.size() == 2
+        mod.scriptModels.size() == 3
```

Comment:
❓ Can we also compare this model with original similarly as in other tests? Otherwise we should remove original model query, but I rather see that we compare the content if we can